### PR TITLE
Home screen

### PR DESCRIPTION
**Pull request by kulloveth**
home

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,18 @@
 Just a personal project playing with the free [SportsDB](https://www.thesportsdb.com/api.php?ref=apilist.fun)
 
 
-## Screen 1 & 2: 
-  <img src="https://user-images.githubusercontent.com/11560987/79799448-77cbd900-8328-11ea-9ffa-53851a2365a9.jpg" width="280"/> <img src="https://user-images.githubusercontent.com/11560987/79799462-7ac6c980-8328-11ea-8f19-f34b43edfd5b.jpg" width="280"/> 
+## Screen 1 & 2:
+<table>
```
Comment (Madonahs) [inline README.md:14]: Extra space :)

File: app/build.gradle [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply plugin:  'dagger.hilt.android.plugin'
 
 android {
     compileSdkVersion 30
     defaultConfig {
         applicationId "com.madonasyombua.sportsdb"
-        minSdkVersion 16
+        minSdkVersion 21
```
Comment (Madonahs) [inline app/build.gradle:10]: Great idea, I however think we can do 24 Lol

File: app/build.gradle [hunk 3]
```diff
@@ -1,16 +1,17 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply plugin:  'dagger.hilt.android.plugin'
```
Comment (Madonahs) [inline app/build.gradle:4]: Perfect

File: app/build.gradle [hunk 4]
```diff
@@ -19,43 +20,79 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    
     buildFeatures{
-        dataBinding = true
+        compose true
```
Comment (Madonahs) [inline app/build.gradle:25]: 💯  this is amazing!!

File: app/src/main/java/com/madonasyombua/sportsdb/app/NetworkModule.kt [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    @Provides
+    @Singleton
+    fun provideOkHttpClient(): OkHttpClient {
+        return OkHttpClient.Builder()
+            .addInterceptor(
+                    HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BODY)
+            )
+            .build()
+    }
+
```
Comment (Madonahs) [inline app/src/main/java/com/madonasyombua/sportsdb/app/NetworkModule.kt:31]: Beautiful

File: app/src/main/java/com/madonasyombua/sportsdb/data/temp.kt [hunk 6]
```diff
@@ -0,0 +1,2 @@
+package com.madonasyombua.sportsdb.data
+
```
Comment (Madonahs) [inline app/src/main/java/com/madonasyombua/sportsdb/data/temp.kt:2]: 👾

File: app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt [hunk 7]
```diff
@@ -1,15 +1,47 @@
...
 
     }
 
 }
+
+@ExperimentalFoundationApi
+@Composable
+fun MyApp(viewModel: HomeViewModel,teamsViewModel: TeamsViewModel) {
+    HomeScreen(viewModel,teamsViewModel = teamsViewModel)
+
```
Comment (Madonahs) [inline app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt:38]: 👾

File: app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt [hunk 8]
```diff
@@ -1,15 +1,47 @@
...
+}
+
+@Preview("Light Theme", widthDp = 360, heightDp = 640)
+@Composable
+fun LightPreview() {
+    MaterialTheme {
+        // MyApp()
```
Comment (Madonahs) [inline app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt:45]: What is `//MyApp()`

File: app/src/main/java/com/madonasyombua/sportsdb/ui/screen/LeagueTabrow.kt [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.madonasyombua.sportsdb.data.remote.model.League
+import com.madonasyombua.sportsdb.ui.theme.mediumSeaGreen
+
+/**
+ * Created by Loveth Nwokike
+ * on 3/17/2021
+ * */
+@Composable
```
Comment (Madonahs) [inline app/src/main/java/com/madonasyombua/sportsdb/ui/screen/LeagueTabrow.kt:16]: 😮  Great Work!! 🤯

File: app/src/main/java/com/madonasyombua/sportsdb/ui/screen/home/HomeScreen.kt [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+                                        teamsViewModel.getTeamsByLeague(it.name)
+                                        TeamsDisplayGrid(viewModel = teamsViewModel)
+                                    }
+                                }
+
+                            }
+                        }
+    }
+}
+
```
Comment (Madonahs) [inline app/src/main/java/com/madonasyombua/sportsdb/ui/screen/home/HomeScreen.kt:60]: 👾

**Review by Madonahs — APPROVED** [review]
I have a few comments otherwise this is great work. Super proud of you! ❤️

File: app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt (hunk 8 above)
Comment (kulloveth) [reply app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt:45]: for previewing, will set it up with dummy data

Comment (Madonahs) [reply app/src/main/java/com/madonasyombua/sportsdb/ui/MainActivity.kt:45]: Great

**Review by Madonahs — COMMENTED** [review]
Great

**Pull request merged**